tests/07-performance.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+source "./helpers.bash"
+
+set -e
+
+current_branch="$(git rev-parse --abbrev-ref HEAD)"
+TEST_NET="cilium"
+
+function cleanup {
+    cd "${dir}/.."
+	git checkout "${current_branch}"
+	git stash pop
+    make
+    sudo make install
+    sudo service cilium-docker restart
+}
+
+trap cleanup EXIT
+
+function install_master {
+    cd "${dir}/.."
+    git stash
+    git checkout origin/master
```

Comment:
Yeah, I got problems myself when there was nothing to stash and when cleaning up I was blindly doing `git stash pop`